### ISSUE TITLE
Trigger functions in app from PM2 CLI

### ISSUE THREAD

**Issue by Unitech** [Unitech/pm2#2509]
We must be able to interact with online processes (eg trigger commands, get results) via PM2 CLI or API:

Expose the ping function from inside your application:

```javascript
var pmx = require('pmx');

pmx.action('ping', function(reply) {
  // Do stuff
  reply('pong');
});
```

Then via the PM2 CLI:

```bash
$ pm2 trigger <app_name|process_id> <action_name> [optional parameters]
```

Example:

```bash
$ pm2 trigger <app_name|process_id> ping
```

```
>>> pm2 trigger index ping
1 processes have received command ping
[index:0]='pong'
```

**Comment by Unitech** [comment]
Available on development branch:

```bash
$ npm install Unitech/pm2#development -g
$ pm2 update
```

**Comment by Unitech** [comment]
published on pm2@2.1.6
